force-app/main/default/lwc/edaSettings/edaSettings.js

```diff
@@ -0,0 +1,8 @@
+import { LightningElement, api } from 'lwc';
+export default class EDASettings extends LightningElement {
+    @api pageReference;
+	LabelReference = {
```

Comment:
Camel Case and spacing
```suggestion
        labelReference = {
```